py-polars/polars/utils/udfs.py

```diff
@@ -159,6 +201,8 @@ def _expr(cls, value: StackEntry, col: str, param_name: str, depth: int) -> str:
             op = value.operator
             e1 = cls._expr(value.left_operand, col, param_name, depth + 1)
             if value.operator_arity == 1:
+                if op.isidentifier():
```

Comment:
this slightly scares me 😄 is there a slightly more robust way?

does this match what's in `NUMPY_MODULE_ALIASES` and `_NUMPY_FUNCTIONS`? If so, could `op` be matched against them directly?

Reply:
just tried replacing this with `if op in _NUMPY_FUNCTIONS` and the tests pass

Reply:
> this slightly scares me 😄 

Lol... it's actually guaranteed to work at this point; however, the _best_ check is probably `op not in _UNARY_OPERATORS`. The only alternative if it fails that check and `operator_arity == 1` is that it _must_ be one of the new synthetic `POLARS_EXPRESSION` ops mapping a function to an expression (I'll add more than numpy matching soon, as well as many more numpy functions, so we don't really want to enumerate all of the explicit matches).

Will tweak accordingly :)